datasetinsights/datasets/synthetic.py

```diff
@@ -170,73 +167,30 @@ def __init__(
     ):
         """
         Args:
-            data_root (str): root directory prefix of dataset
-            manifest_file (str): path to a manifest file. Use this argument
-                if the synthetic data has already been downloaded. If the
-                synthetic dataset hasn't been downloaded, leave this argument
-                as None and provide the run_execution_id and auth_token and
-                this class will download the dataset. For more information
-                on Unity Simulations (USim) please see
-                https://github.com/Unity-Technologies/Unity-Simulation-Docs
+            data_path (str): Directory of the dataset
             transforms: callable transformation that applies to a pair of
             capture, annotation.
             version(str): synthetic dataset schema version
             def_id (int): annotation definition id used to filter results
-            run_execution_id (str): USim run execution id, if this argument
-                is provided then the class will attempt to download the data
-                from USim. If the data has already been downloaded locally,
-                then this argument should be None and the caller should pass
-                in the location of the manifest_file for the manifest arg.
-                For more information
-                on Unity Simulations (USim) please see
-                https://github.com/Unity-Technologies/Unity-Simulation-Docs
-            auth_token (str): usim authorization token that can be used to
-                interact with usim API to download manifest files. This token
-                is necessary to download the dataset form USim. If the data is
-                already stored locally, then this argument can be left as None.
-                For more information
-                on Unity Simulations (USim) please see
-                https://github.com/Unity-Technologies/Unity-Simulation-Docs
             random_seed (int): random seed used for splitting dataset into
                 train and val
         """
-        dataset_directory = os.path.join(data_root, const.SYNTHETIC_SUBFOLDER)
         if (
-            os.path.isdir(dataset_directory)
-            and glob.glob(f"{dataset_directory}/**/*.png")
-            and glob.glob(f"{dataset_directory}/**/*.json")
+            os.path.isdir(data_path)
+            and glob.glob(f"{data_path}/**/*.png")
```

Comment:
PNG files are not required for Synthetic dataset. We could change image format to jpeg or the dataset might include only LIDAR files such as `*.pcd`

Reply:
Can we change this in a follow up PR?